packages/graphql-transformers-e2e-tests/package.json

```diff
@@ -22,6 +22,7 @@
     "build-tests": "yarn tsc --build tsconfig.tests.json"
   },
   "dependencies": {
+    "amplify-cli-core": "^2.5.0",
```

Comment:
Same comment as above. We should avoid this dependency.

Reply:
updated